Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -123,7 +123,7 @@ private function grabAttribsBeforeMethodToken(Tokens $tokens, $index)
      * @param Tokens $tokens Tokens collection
      * @param int    $index  token index
      *
-     * @return array array of grabbed attributes
+     * @return array<string, Token|null> array of grabbed attributes
```

Comment:
> this syntax is not supported by our current fixers

all is green


> it's also not widely accepted as a standard

accepted by PSR5 proposal


> the same way that PSR-5 says don't use `@var`, but we all do anyway

what?
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#824-var